package.json

```diff
@@ -80,7 +80,6 @@
     "react-transition-group": "^2.2.1",
     "sinon": "^7.3.2",
     "typescript": "^3.2.2",
-    "uglifyjs-webpack-plugin": "^1.2.5",
```

Comment:
Do we not need a new package for terser?

Reply:
@stefceror Oh good catch, I wonder why this worked. It must have been able to resolve terser-webpack-plugin somehow, maybe since webpack has it as a dependency? I'll include it explicitly as a dependency though

Reply:
Thanks, I think it's best to be explicit about dependencies.